py/farm_ng/canbus/packet.py

```diff
@@ -491,3 +492,133 @@ def is_button_pressed(self, button: PendantButtons) -> bool:
 
     def __str__(self):
         return "x {:0.3f} y {:0.3f} buttons {}".format(self.x, self.y, self.buttons)
+
+
+class BugDispenserRpdo3(Packet):
+    """Bug dispenser rate in m/drop (request) sent to the Amiga dashboard."""
+
+    cob_id = 0x400
```

Comment:
Define in one place and use in encode / decode
```suggestion
    cob_id = 0x400
    scale = 10.0
```